src-electron/db/zap-schema.sql

```diff
@@ -182,6 +182,7 @@ CREATE TABLE IF NOT EXISTS "COMMAND" (
   "RESPONSE_NAME" integer,
   "RESPONSE_REF" integer,
   "IS_DEFAULT_RESPONSE_ENABLED" integer,
+  "IS_LARGE_MESSAGE" integer,
```

Comment:
Is this the only quality tag attribute which is currently being added by CSA xml? Wondering if there are additional ones and if it is quicker to add those right now while you are at it.